doc/API.md

```diff
@@ -22,6 +22,7 @@ Display general information about the current daemon state.
 | ------------- | ------- | ------------------------------------------------------------- |
 | `blockheight` | integer | Current block height                                          |
 | `network`     | string  | Answer can be `mainnet`, `testnet`, `regtest`                 |
+| `synced`      | bool    | True if the daemon is synced with the blockchain              |
```

Comment:
I'm not sure i can yet commit to a specific semantic API (such as a block height or anything) but a progress percentage may make sense here instead, at least.